packages/dashboard/stories/components/Wallet/ReceiveDialog.tsx

```diff
@@ -0,0 +1,18 @@
+import { story } from '@dimensiondev/maskbook-storybook-shared'
+import { ReceiveDialog as C } from '../../../src/pages/Wallets/components/ReceiveDialog'
+import { action } from '@storybook/addon-actions'
+
+const { meta, of } = story(C)
+
+export default meta({
+    title: 'Components/Wallet/Receive Dialog',
+})
+
+export const ReceiveDialog = of({
```

Comment:
Cannot read property 'displayName' of undefined

In storybook

Reply:
I review and fix the component, but it still is in. What is puzzling is that the component import from `ReceiveDialog` console log undefined. I checked the storybook issue for solutions to the same problem, but none of them solved it @Jack-Works